src/controller/FileBrowserController.vala

```diff
@@ -28,7 +28,7 @@ namespace BrickManager {
     public class FileBrowserController : Object, IBrickManagerModule {
         const string PARENT_DIRECTORY_TEXT = "../";
         const string INITAL_DIRECTORY = "/home/robot";
-        const string USER_NAME = "ev3dev";
+        const string USER_NAME = "robot";
```

Comment:
We're trying to set the permissions to "ev3dev:" and that fails for sure. If robot is a member of tty we can maybe skip the whole chown business, let me retry.